iothub_client/devdoc/iothubclient_c_library.md

```diff
@@ -501,6 +501,13 @@ const char* x509privatekey =
 ```
 
 - "CypherSuite" - only available when OpenSSL is used. value is a pointer to a null terminated string that contains a list in the format specified by [SSL_set_cipher_list](https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_cipher_list.html).
+
```

Comment:
I'm not following the flow here.  Are "Engine" and "x509PrivateKeyType" other OPTION_OPENSSL_CIPER_SUITE values??  If not, perhaps they need their own header? Not sure.

Reply:
Good catch! Fixed.